packages/client/modules/email/components/SummaryEmail/ExportToPDF.tsx

```diff
@@ -0,0 +1,57 @@
+import withAtmosphere, {WithAtmosphereProps} from 'parabol-client/decorators/withAtmosphere/withAtmosphere'
+import {PALETTE} from 'parabol-client/styles/paletteV3'
+import withMutationProps, {WithMutationProps} from 'parabol-client/utils/relay/withMutationProps'
+import React, {Component} from 'react'
+import {ExternalLinks} from '../../../../types/constEnums'
+
+interface Props extends WithAtmosphereProps, WithMutationProps {}
+
+const label = 'Export to PDF'
+
+const tdStyle = {
+  color: PALETTE.SLATE_700,
+  fontSize: '13px',
+  paddingTop: 32,
+}
+
+const labelStyle = {
+  paddingLeft: 8
+}
+const imageStyle = {
+  verticalAlign: 'middle'
+}
+
+const linkStyle = {
+  height: '40px',
+  padding: '12px 10px 0px 10px',
+  cursor: 'pointer',
+  width: 'fit-content',
+}
```

Comment:
-1 when working with inline styles, please use the proper type, for example, `const linkStyle: React.CSSProperties`